Makefile

```diff
@@ -0,0 +1,31 @@
+include linting.mk
+
+export GO111MODULE=on
+
+.PHONY: ci
```

Comment:
can we add a simple GHA workflow?

Reply:
added very basic linting, testing and building